login.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <main class="login-main">
+    <div class="login-container">
+      <div class="login-image-container row">
+        <img src="assets/logo-wolox.png" alt="Logo Wolox" class="login-image-logo">
+      </div>
+      <form action="list.html" class="login-form">
+        <div class="column login-email-container">
+          <label for="login-email" class="login-label text-5">Email</label>
+          <input type="email" id="login-email" class="login-input border-radius-5">
+        </div>
+        <div class="column login-password-container">
```

Comment:
I think it would be easier to set column on your form tag and that way you don't have to use a container for every input you have. Also, you can a set `margin-bottom` inside of your `login-input` class